charts/cray-certmanager/templates/cert-manager/rbac.yaml

```diff
@@ -0,0 +1,31 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cert-manager
+rules:
+  - apiGroups: [""]
+    resources: [configmaps]
+    verbs: [get, patch]
+  - apiGroups: ["", "apps"]
+    resources: ["pods", "deployments"]
```

Comment:
I don't think this job needs to do anything with pods, it's only dealing with a deployment.